Common/src/interpolation_structure.cpp

```diff
@@ -688,19 +689,24 @@ void CNearestNeighbor::Set_TransferCoeff(CConfig **config) {
 
         for (iProcessor = 0; iProcessor < nProcessor; iProcessor++) {
           for (jVertex = 0; jVertex < MaxLocalVertex_Donor; jVertex++) {
+           
             Global_Point_Donor = iProcessor*MaxLocalVertex_Donor+jVertex;
-
-            Coord_j = &Buffer_Receive_Coord[ Global_Point_Donor*nDim];
-
-            dist = PointsDistance(Coord_i, Coord_j);
-
-            if (dist < mindist) {
-              mindist = dist; pProcessor = iProcessor; pGlobalPoint = Buffer_Receive_GlobalPoint[Global_Point_Donor];
+            
+            if (Buffer_Receive_GlobalPoint[Global_Point_Donor] != -1){
+              
+              Coord_j = &Buffer_Receive_Coord[ Global_Point_Donor*nDim];
+              
+              dist = PointsDistance(Coord_i, Coord_j);
+              
+              if (dist < mindist) {
+                mindist = dist; pProcessor = iProcessor; 
+                pGlobalPoint = Buffer_Receive_GlobalPoint[Global_Point_Donor];                
+              }
```

Comment:
Thanks again @talbring @rsanfer (and everyone involved I might not know about) for finding & fixing this bug!